### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularForgotPassword_NonTextContrast: The Color contrast ratio of the focus indicator of the "Reset Password" button is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42335]
![42335_A11y_ NET Core_ASP NETCore_AngularForgotPassword_NonTextContrast](https://user-images.githubusercontent.com/86710491/175480119-21cfbc53-6d9c-401b-8817-b090266fccb4.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopWeb;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. Open command prompt.
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Hit enter on login link
10. Tab and navigate to "forgot your password?" link and press enter.
11. Turn on Color contrast analyzer and check that The Color contrast ratio of "Reset Password" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Reset Password" button is less than 3:1.

**This issue is observed throughout the application for all the "Reset Password" buttons.**

### Expected:
The Color contrast ratio of the focus indicator of the  "Reset Password" button should be more than or equal to 3:1.

### User Impact:
Low vision users wont be able to see the focus indicator.

**Comment by HaoK** [comment]
@merriemcgaw are we able to resolve this contrast issue as external (bootstrap defaults) since the focus color here is the default in bootstrap, and we are trying to generally use the standard look and feel of bootstrap in identity UI?

**Comment by merriemcgaw** [comment]
@HaoK Is there a way to change it on our own? If there is, I'd highly recommend it, along with filing an issue with bootstrap that their focus indicator is not accessible. We're required to be accessible if it is at all possible. If there's no way to fix it on our end, we can work together to mark this as external and try to get an exception, though I'm doubtful we'd get one for focus indicators.

**Comment by HaoK** [comment]
Okay I think we hardcoded some colors before, so I guess I can just pick a new color for focus that satisfies the ratio

**Comment by HaoK** [comment]
Fixed in https://github.com/dotnet/aspnetcore/pull/42840

**Comment by Truptik12** [comment]
@HaoK 

Verified the bug, the bug still reproduces.

**Comment by HaoK** [comment]
Can you attach a new screenshot?  I thought we updated thing sto have a solid black border, i.e. https://user-images.githubusercontent.com/6537861/180089982-c1c7b8b3-262f-4bb3-b174-c2eb26318890.png

**Comment by merriemcgaw** [comment]
@Truptik12 did you install the latest SDK in your testing from here: https://dotnet.microsoft.com/en-us/download/dotnet/7.0

@HaoK was the update in Preview 6 for sure? Or do we need to wait for Preview 7?

**Comment by HaoK** [comment]
Ah right, this is fixed in aspnetcore-main which would be rc1 I think

**Comment by HaoK** [comment]
Should be available in the https://github.com/dotnet/installer nightlies for main

**Comment by Truptik12** [comment]
Verified the bug after installing the https://github.com/dotnet/installer , the bug still reproduces.
![image](https://user-images.githubusercontent.com/86710491/182097433-20d41a27-534a-4134-ac0e-0131488b38db.png)

**Comment by HaoK** [comment]
@Truptik12 can you confirm the version you are running for me by including the contents of your csproj for app?

**Comment by HaoK** [comment]
Okay I think I see the issue, reassigning back to myself

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/spa-templates/pull/73

**Comment by merriemcgaw** [comment]
@Truptik12  can you please validate this with the latest .NET Preview 7 SDK?

**Comment by HaoK** [comment]
I think the fixes for these are only in main/rc1

**Comment by merriemcgaw** [comment]
@Truptik12 that means you will need to download the latest RC1 build from the first column of [this table](https://github.com/dotnet/installer#table). That should give you the updated templates for all the recent fixes.

**Comment by Truptik12** [comment]
@merriemcgaw 

We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version and the bug.
![image](https://user-images.githubusercontent.com/86710491/185577237-7390b02a-0cc9-4305-a972-462d78846b6b.png)

![image](https://user-images.githubusercontent.com/86710491/185577609-d784dfa0-272b-4570-bc36-1e6427150a91.png)


Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.

**Comment by HaoK** [comment]
Okay the fix might not have made it to the installers yet for the angular/react templates:

They went in on via a dependency update https://github.com/dotnet/aspnetcore/pull/43289 4 days ago.

I see the changes in the blazorserver site.css, but you might not be able to verify those until https://github.com/dotnet/aspnetcore/pull/43375 makes it up to the installers since that bug blocks things.

**Comment by merriemcgaw** [comment]
@HaoK can you verify that #43375 has made it to the installers at this point (I _think_ it should have) so that @Truptik12 can validate?

**Comment by Truptik12** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.

![image](https://user-images.githubusercontent.com/86710491/188159743-6b2e9020-ad70-4304-8289-b3bd3e33c2d6.png)

**Comment by merriemcgaw** [comment]
Thanks @Truptik12. @HaoK can we find out when this fix is going to make it to installers/SDK for RC2? We are cutting it very close.

**Comment by merriemcgaw** [comment]
@Truptik12 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached.

![image](https://user-images.githubusercontent.com/86710491/192524701-27b6f235-9d37-41ec-8960-ca76cb72fcf4.png)
